openml/runs/run.py

```diff
@@ -395,7 +394,7 @@ def extract_parameters(_flow, _flow_dict, component_model,
             expected_components = set(_flow.components)
             model_parameters = set([mp for mp in component_model.get_params()
                                     if '__' not in mp])
-            if len((expected_parameters | expected_components) ^ model_parameters) != 0:
+            if(len((expected_parameters | expected_components) ^ model_parameters) != 0):
```

Comment:
Is there a reason to change these? If not, please revert to make this file not look like it was edited/affected by this PR.